contracts/BitGuildMarketplace.sol

```diff
@@ -0,0 +1,355 @@
+pragma solidity ^0.4.22;
+
+import "./BitGuildToken.sol";
+import "./BitGuildAccessAdmin.sol";
+import "./BitGuildWhitelist.sol";
+import "./BitGuildFeeOracle.sol";
+import "./SafeMath.sol";
+
+
+// @dev Interface for ERC-721 NFT
+contract ERC721Interface {
```

Comment:
Move out in a separate file, import file. We will replace it later with a proper ERC721 impl when we will merge the repos.